collection/slice.go

```diff
@@ -0,0 +1,25 @@
+package collection
+
+// EqualValuesI returns true if the two slices are values equal ignoring the order
+func EqualValuesI(a, b []interface{}) bool {
```

Comment:
Just wondering if we have use case for an interface parameter? Because I have some issues:

```
ts1 := "a"
ts11 := "b"
ts2 := "a"
ts22 := "b"
t := []interface{}{struct{ X, Y *string }{&ts1, &ts11}}
t2 := []interface{}{struct{ X, Y *string }{&ts2, &ts22}}
fmt.Println(EqualValuesI(t, t2)) // returns false

v1 := []interface{}{
	[]int{1, 2},
}
v2 := []interface{}{
	[]int{1, 2},
}
fmt.Println(EqualValuesI(v1, v2)) // runtime error: hash of unhashable type []int
```